### PR TITLE
update linkerd to 1.0.0, linkerd-viz to 0.1.0

### PR DESCRIPTION
**Pull request by siggy**

### CODE REVIEW

File: repo/packages/L/linkerd/10/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "3.0",
+  "name": "linkerd",
+  "version": "1.0.0-0.1",
+  "scm": "https://github.com/linkerd/linkerd/",
+  "description": "A dynamic linker for microservices. Install, set your http proxy to $HOST:4140, and connect to your services via app name. linkerd will handle Marathon service discovery for you.",
+  "maintainer": "linkerd-users@googlegroups.com",
```
Comment (ryadav88) [inline repo/packages/L/linkerd/10/package.json]: `maintainer` field should be a mailing list. 
I see it was `hello@buoyant.io` in the last version

File: repo/packages/L/linkerd-viz/3/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "3.0",
+  "name": "linkerd-viz",
+  "version": "0.1.0-0.1",
+  "scm": "https://github.com/linkerd/linkerd-viz/",
+  "description": "A visualization dashboard for linkerd.",
+  "maintainer": "linkerd-users@googlegroups.com",
```
Comment (ryadav88) [inline repo/packages/L/linkerd-viz/3/package.json]: see comment about `maintainer` property below.

File: repo/packages/L/linkerd-viz/3/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "website": "https://linkerd.io",
+  "tags": ["linkerd", "namerd", "buoyant", "loadbalancer", "service-discovery", "proxy", "microservices", "rpc", "linkerd-viz", "dashboard", "visualization", "metrics"],
+  "preInstallNotes": "This DC/OS Service is currently in preview. We recommend a minimum of 1.0 CPU and 512 MB of RAM available for linkerd-viz.",
```
Comment (ryadav88) [inline repo/packages/L/linkerd-viz/3/package.json]: add a link to dcos/examples at the end of `preInstallNotes`. 
Also, please update the example to add 1.9 instructions (see other examples): https://github.com/dcos/examples/tree/master/linkerd

File: repo/packages/L/linkerd/10/package.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  "website": "https://linkerd.io",
+  "tags": ["linkerd", "namerd", "buoyant", "loadbalancer", "service-discovery", "proxy", "microservices", "rpc"],
+  "preInstallNotes": "This DC/OS Service is currently in preview. We recommend a minimum of 0.25 CPUs and 256 MB of RAM available for linkerd. We also recommend running linkerd on every host. To ensure this, set instances to the number of nodes in your cluster. Installation Documentation: https://github.com/dcos/examples/tree/master/1.8/linkerd",
```
Comment (ryadav88) [inline repo/packages/L/linkerd/10/package.json]: change https://github.com/dcos/examples/tree/master/1.8/linkerd to https://github.com/dcos/examples/tree/master/linkerd

**Comment by ryadav88** [comment]
Tested it on 1.9. LGTM apart from the requested changes above.

File: repo/packages/L/linkerd/10/package.json (hunk 1 above)
Comment (siggy) [inline repo/packages/L/linkerd/10/package.json]: @ryadav88 `linkerd-users@googlegroups.com` is our primary mailing list, is that ok?

Comment (ryadav88) [inline repo/packages/L/linkerd/10/package.json]: can this mailing list be used by customers looking for linkerd support? I'm assuming `inkerd-users@googlegroups.com` is an open user mailing list.

Comment (siggy) [inline repo/packages/L/linkerd/10/package.json]: yup. that mailing list is our primary asynchronous user support channel.

File: repo/packages/L/linkerd-viz/3/package.json (hunk 3 above)
Comment (siggy) [inline repo/packages/L/linkerd-viz/3/package.json]: https://github.com/dcos/examples/pull/150

**Pull request merged**